### PR TITLE
Adding a whitelist of package files

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
Also uppercasing the changelog file. It's uppercase everywhere ^^'

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "contributors": [
     "Yuri Astrakhan <yurik@wikimedia.org>"
   ],
+  "files": ["index.js"],
```
Comment (adamwight) [inline package.json:9]: Want to add README.md too?  "README" is pulled in by default but I don't think that the .md files will be unless listed here.

Comment (WMDE-Fisch) [reply package.json:9]: npm publish now includes.
```
npm notice 725B  LICENSE     
npm notice 8.0kB index.js    
npm notice 559B  package.json
npm notice 666B  CHANGELOG.md
npm notice 3.2kB README.md 
```
:-)

Comment (WMDE-Fisch) [reply package.json:9]: ( seems to be clever enough )

**Comment by adamwight** [comment]
Thanks, and sorry that I didn't do this experimental test locally...

**Pull request merged**